packages/app/src/cli/models/app/app.ts

```diff
@@ -314,11 +314,23 @@ export class App implements AppInterface {
     const scopes = getPathValue<string>(configuration, 'access_scopes.scopes')
     const useLegacyInstallFlow = getPathValue<boolean>(configuration, 'access_scopes.use_legacy_install_flow')
     const redirectUrls = getPathValue<string[]>(configuration, 'auth.redirect_urls')
+    const directApiMode = getPathValue<string>(configuration, 'access.admin.direct_api_mode')
+    const embeddedAppDirectApiAccess = getPathValue<string>(
+      configuration,
+      'access.admin.embedded_app_direct_api_access',
+    )
     return {
       ...(scopes || useLegacyInstallFlow
         ? {access_scopes: {scopes, use_legacy_install_flow: useLegacyInstallFlow}}
         : {}),
       ...(redirectUrls ? {auth: {redirect_urls: redirectUrls}} : {}),
+      ...(directApiMode
+        ? {
+            access: {
```

Comment:
TODO: Test direct_api_mode with falsy embedded_app_direct_api_access, and vice versa.